hawkular-inventory-bus-api/src/main/java/org/hawkular/inventory/bus/api/DataEntityEvent.java

```diff
@@ -29,8 +30,8 @@
     public DataEntityEvent() {
     }
 
-    public DataEntityEvent(Action.Enumerated action, DataEntity object) {
-        super(action, object);
+    public DataEntityEvent(Action.Enumerated action, Tenant tenant, DataEntity object) {
```

Comment:
What do you need the tenant for? I haven't spotted anything in this PR that would require it...